SPECS/qemu-kvm/qemu-kvm.spec

```diff
@@ -122,36 +125,54 @@ chmod 755 %{buildroot}%{_bindir}/qemu
 %{_bindir}/qemu-nbd
 
 %changelog
+* Tue Oct 27 2020 Henry Li <lihl@microsoft.com> - 4.2.0-14
```

Comment:
Nit: remove the "-" between "microsoft.com>" and "4.2.0-14".

Reply:
@niontive This is actually our preferred style now, as enforced by the linter.

Reply:
Gotcha- I was just considering the older changelog for this spec. Will note for the future!